ui/console/pom.xml

```diff
@@ -79,4 +104,39 @@
             </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>dev</id>
```

Comment:
dev profile is default. only if you specify -Prelease, the stuff is deleted. if you specify nothing, the downloaded node stuff stays in target directory.